tests/data/challenge_response.py

```diff
@@ -17,7 +17,7 @@
                             "description": "Excepturi eligendi minus modi delectus doloreasperiores voluptatem. \
```

Comment:
I think you also need to add 1 or 2 test cases. I am not sure though but can you please check if all the cases are covered by test cases.